src/main/java/org/snomed/snowstorm/core/data/domain/jobs/ExportConfiguration.java

```diff
@@ -48,6 +48,12 @@ public class ExportConfiguration {
 	@Schema(description = "If refsetIds are included, this indicates that the export will be a refset-only export.")
 	private Set<String> refsetIds;
 
+	@Schema(defaultValue = "PENDING")
+	private String status;
```

Comment:
Whenever an export job is created, it will have the "PENDING" status.

Reply:
@dmcgihtsdo We can use ExportStatus as the data type.